ibverbs-sys/build.rs

```diff
@@ -8,6 +8,11 @@ fn main() {
     println!("cargo:rustc-link-search=native={manifest_dir}/vendor/rdma-core/build/lib");
     println!("cargo:rustc-link-lib=ibverbs");
 
+    if env::var("CARGO_FEATURE_EFA").is_ok() && Path::new("/opt/amazon/efa/lib/").exists() {
```

Comment:
since this is now feature gated, i would remove:
Path::new("/opt/amazon/efa/lib/").exists()